catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -419,6 +419,15 @@
         <feature>catalog-core</feature>
     </feature>
 
+    <feature name="catalog-metacard-enrichment" install="auto" version="${project.version}"
+             description="Conditional, rule-based enhancement and modification to metacards">
+        <bundle dependency="true">mvn:ddf.catalog.core/catalog-core-api/${project.version}</bundle>
+        <bundle dependency="true">mvn:org.apache.shiro/shiro-core/1.2.4</bundle>
+        <bundle>mvn:ddf.platform/platform-filter-clientinfo/${project.version}</bundle>
+        <!--This feature is choking on this bundle? -->
```

Comment:
I specified an interface with `org.codice.ddf.*` when it really should just be `ddf.*` in my blueprint.